tools/validate-links.mjs

```diff
@@ -0,0 +1,45 @@
+import fs from 'fs-extra';
```

Comment:
Here's the script that validates links. We're now running on a new-enough version of Node that we can use `.mjs` files with native ES modules!